.github/workflows/benchmark.yml

```
@@ -0,0 +1,80 @@
+name: Benchmark
+# Run on push only for master, if not it will trigger push & pull_request on PRs at the same time
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      # Any update here needs to be done for
+      # - `pull_request` see below
+      # - https://github.com/prisma/prisma/blob/master/src/.buildkite/test/buildkite-entry.sh
+      # - https://github.com/prisma/prisma/blob/master/src/.buildkite/publish/buildkite-entry.sh
+      - "*.md"
+      - ".vscode/**"
+      - "docs/**"
+      - "examples/**"
+      - "LICENSE"
+      - "src/.buildkite/**"
+      - "src/.dockerignore"
+      - "src/scripts/ci/publish.ts"
+      - "src/graphs/**"
+  pull_request:
+    paths-ignore:
+      # Any update here needs to be done for
+      # - `push`see before
+      # - https://github.com/prisma/prisma/blob/master/src/.buildkite/test/buildkite-entry.sh
+      # - https://github.com/prisma/prisma/blob/master/src/.buildkite/publish/buildkite-entry.sh
+      - "*.md"
+      - ".vscode/**"
+      - "docs/**"
+      - "examples/**"
+      - "LICENSE"
+      - "src/.buildkite/**"
+      - "src/.dockerignore"
+      - "src/scripts/ci/publish.ts"
+      - "src/graphs/**"
+jobs:
+  benchmark:
+    name: all
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: ~/.pnpm-store
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "12"
+
+      - run: bash .github/workflows/setup.sh
+        env:
+          CI: true
+          SKIP_GIT: true
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+      #  This is required as setup.sh can modify pnpm-lock.yml
+      - run: rm -f src/pnpm-lock.yaml
+      - name: Run benchmarks
+        run: yarn bench
```

```suggestion
        run: pnpm run bench
```
We use `pnpm` in CI actions, maybe better to use the same everywhere